### DISCUSSION TITLE
Can we just use other License by replacing the boilerplate file under folder hack?

### DISCUSSION THREAD

**Discussion by Jaxx4Fun** [kubernetes-sigs/kubebuilder#2938] [Support] [answered]
Is it mandatory that the code generated by kubebuilder should be under Apache License 2.0 as kubebuilder does?

**Comment by camilamacedo86** [comment, marked as answer]
Yes, you can see. That you have by default 2 options:

https://github.com/kubernetes-sigs/kubebuilder/blob/1e09811e0e8489bf01c74a78ed32eb749fa05cf5/pkg/plugins/golang/v3/init.go#L87-L88

Also, you can customize the boollerplate.txt that is scaffolded by you.

If you see how to improve on this feature and would like to contribute, please feel free to push a PR for example to add an option for your init with some customization or another license.

**Comment by camilamacedo86** [comment]
Closing since it is answered.
